Source/Scene/Cesium3DTileset.js

```diff
@@ -227,6 +229,24 @@ define([
         this._maximumNumberOfLoadedTiles = defaultValue(options.maximumNumberOfLoadedTiles, 256);
         this._styleEngine = new Cesium3DTileStyleEngine();
 
+        /**
+         * Defines how per-feature colors blend with the source colors.
```

Comment:
This should be more explicit: per-feature colors _from the Cesium API or declarative styling_, and source colors _from the orignial feature, e.g., glTF material or per-point color in the tile._